community/blog/2020.10-Introducing-message-handling-with-Azure-Service-Bus-message-influence/README.md

```diff
@@ -0,0 +1,106 @@
+# Introducing message handling with Azure Service Bus message influence in v0.5
```

Comment:
It might be me, but when I read the title, I have no idea what the paragraph/chapter will talk about ?  What is meant with 'message influence' ?

Reply:
Yes, you may be right. I mostly took that from the [original issue](https://github.com/arcus-azure/arcus.messaging/issues/118) that used these words.
It's about calling the specific Azure Service Bus message operations during the message handling. Could you give some alternative, then? "Control Azure Service Bus message during message handling?"

Reply:
@tomkerkhove , should I change the title here? Any suggestions?